### PR TITLE
INSTA-17267-Document Synthetic Metrics

### PR DESCRIPTION
**Pull request by kmcormick**
Add Synthetic Metrics documentation

### CODE REVIEW

File: spec/descriptions/tagSyntheticMetrics.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  * The granularity should not be set too small relative to the `windowSize` to avoid creating an excessively large number of data points (max 600)
+  * The granularity values are the same for all metrics
+
+**pagination** if you use pagination, fix the timeFrame for the retrieved metrics
+1. *page* select the page number you want to retrieve
+2. *pageSize* set the number of Synthetic test results you want to return with one query
+
+**groups** You can group test results by one or more tags.
+1. *groupbyTag* use the metric or tag name, e.g. "synthetic.applicationId", to group by its value
+2. *groupbyTagSecondLevelTag* optional, used to further qualify values when the *groupbyTag* is an array.
```
Comment (carolzhang65) [inline spec/descriptions/tagSyntheticMetrics.md:46]:
```suggestion
2. *groupbyTagSecondLevelTag* is optional. It is used to further qualify values when the *groupbyTag* is an array.
```

**Review by carolzhang65 — APPROVED** [review]
It looks good except one minor suggestion. Thanks.

File: spec/descriptions/tagSyntheticMetrics.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+```
+  "groups": [
+    {
+      "groupbyTag": "synthetic.tags",
+      "groupbyTagSecondLevelTag": "region"
+    }
+  ]
+```
+
+**includeAggregatedTestIds** Optionally use when not grouping by testId.  Specifying *true* will return a list of tests that were included in the aggregated result for each row returned.
```
Comment (andreachiavegatto) [inline spec/descriptions/tagSyntheticMetrics.md:61]: Should this be "Optionally used"?

**Review by andreachiavegatto — APPROVED** [review]
lgtm

File: spec/descriptions/tagSyntheticMetrics.md (hunk 1 above)
Comment (kmcormick) [reply spec/descriptions/tagSyntheticMetrics.md:46]: done.  thanks...

File: spec/descriptions/tagSyntheticMetrics.md (hunk 2 above)
Comment (kmcormick) [reply spec/descriptions/tagSyntheticMetrics.md:61]: yes, thanks...

**Pull request merged**